### PR TITLE
MiaoYan 0.6.0

### PR DESCRIPTION
**Pull request by JounQin**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/miaoyan.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cask "miaoyan" do
+  version "0.5.1"
+  sha256 "fd0a8c14bba2bfd488771a351b103a95759694cb5a73e7e33dc4e7f5b1ef3ab0"
+
+  url "https://github.com/tw93/MiaoYan/releases/download/V#{version}/MiaoYan.dmg",
+      verified: "github.com/tw93/MiaoYan"
+  name "MiaoYan"
+  desc "Markdown editor"
```
Comment (JounQin) [inline Casks/miaoyan.rb:8]: @bevanjkay I'm not sure why you made this change? Is there anything I missed?

Comment (bevanjkay) [reply Casks/miaoyan.rb:8]: @JounQin The previous description did not follow the guidelines for the `desc` stanza - https://docs.brew.sh/Cask-Cookbook#stanza-desc

Comment (JounQin) [reply Casks/miaoyan.rb:8]: Thanks for pointing it to me.

**Comment by JounQin** [comment]
@bevanjkay Any chance to merge then?

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**